docs/3.0/deploy/infrastructure-concepts/work-pools.mdx

```diff
@@ -423,9 +423,6 @@ Learn more about [customizing job variables](/3.0/deploy/infrastructure-concepts
     capabilities in secure environments. For example, the `kubernetes` worker type allows users to specify a custom job template  
     to configure the manifest that workers use to create jobs for flow execution.
 
-    For more information and advanced configuration examples, see the [Kubernetes Worker](https://prefecthq.github.io/prefect-kubernetes/worker/) 
-    documentation.
-
```

Comment:
we can link [here](https://docs.prefect.io/integrations/prefect-kubernetes/index) instead of totally removing this right?

Reply:
I didn't find a good place to link to. Do you have a rec?

Reply:
the link I suggested is in my previous comment

https://docs.prefect.io/integrations/prefect-kubernetes/index

Reply:
We link to those docs at the end of that page and the notes isn't for Kubernetes specifically, so I think just having one link in the notes - to the overriding job variables docs, is best. If you feel strongly, can keep that link in.